parsers/US_AK_SAP.py

```diff
@@ -0,0 +1,64 @@
+from datetime import datetime, timezone
+from logging import Logger, getLogger
+from typing import List, Optional, Union
+
+from requests import Session
+
+from electricitymap.contrib.lib.models.event_lists import ProductionBreakdownList
+from electricitymap.contrib.lib.models.events import ProductionMix
+from electricitymap.contrib.lib.types import ZoneKey
+from parsers.lib.exceptions import ParserException
+
+PARSER_NAME = "US_AK_SAP.py"
+
+SOURCE = "seapahydro.org"
+DATA_URL = "https://seapahydro.org/api/scada/index"
+
+
+def get_value(data: dict, key: str) -> float:
+    return float(data[key]["text"])
+
+
+def fetch_production(
+    zone_key: ZoneKey,
+    session: Session = Session(),
+    target_datetime: Optional[datetime] = None,
+    logger: Logger = getLogger(__name__),
+) -> Union[List[dict], dict]:
+    if target_datetime is not None:
+        raise ParserException(
+            PARSER_NAME, "This parser is not yet able to parse past dates", zone_key
+        )
+
+    res = session.get(DATA_URL)
+    data = res.json()
+
+    data = {
+        "seapa_total": get_value(data, "ss_mw"),  # 2 hydro plants owned by SEAPA
+        "ktn_hydro": get_value(
+            data, "ktn_hydro_mw"
+        ),  # hydro from pre-existing Ketchikan plants
+        "ktn_diesel": get_value(data, "ktn_diesel_mw"),  # backup Ketchikan diesel
+    }
+
+    production_mix = ProductionMix(gas=data["ktn_diesel"])
+    production_mix.add_value("hydro", data["seapa_total"], True)
+    production_mix.add_value("hydro", data["ktn_hydro"], True)
```

Comment:
```suggestion
    production_mix = ProductionMix()
    production_mix.add_value("gas", gas=data["ktn_diesel"], True)
    production_mix.add_value("hydro", data["seapa_total"], True)
    production_mix.add_value("hydro", data["ktn_hydro"], True)
```
This could also be done it a loop but since it's only 3 elements we can keep it like this for now.